app/common/model/src/main/java/io/syndesis/common/model/integration/IntegrationOverview.java

```diff
@@ -60,6 +61,10 @@ default IntegrationBulletinBoard getBoard() {
 
     Optional<String> getUrl();
 
+    Optional<String> getUrlTo3scale();
+
+    Optional<List<String>> getExposureOptions();
```

Comment:
No need for `Optional` with collections

```suggestion
    Set<String> getExposureOptions();
```